src/Purpose.html

```diff
@@ -5563,79 +5563,139 @@
+&lt;speech.tiffany&gt;Let go!&lt;/speech&gt;
 
-&quot;You alright %Tiffany%?&quot;, I ask.
+&lt;action&gt;She says.&lt;/action&gt;
+&lt;action&gt;I release my grip and she disappears out of view onto the patio. &lt;/action&gt;
+&lt;action&gt;I wait a moment for her to say something. &lt;/action&gt;
+
+&lt;speech.sarah&gt;You alright %Tiffany%?&lt;/speech&gt;
+&lt;action&gt;I ask.&lt;/action&gt;
 
 &lt;% if (!window.story.getChoice(&quot;Chapter1&quot;, &quot;StrangerDied&quot;)) { %&gt;
 
-&quot;Yeah you good down there?&quot;, the stranger adds.
+&lt;resetcharacter.two/&gt;
+&lt;character.two&gt;Unknown&lt;/character&gt;
+
+&lt;speech.unknown&gt;Yeah you good down there?&lt;/speech&gt;
+&lt;action&gt;the stranger adds.&lt;/action&gt;
 
 &lt;% } %&gt;
 
-There&#39;s a moment&#39;s pause.
+&lt;character.one.pain.neutral.neutral-backpack&gt;Sarah&lt;/character&gt;
+&lt;character.two.confused.sad.neutral-backpack&gt;Tiffany&lt;/character&gt; 
+
+&lt;action&gt;There&#39;s a moment&#39;s pause. &lt;/action&gt;
+
+&lt;speech.sarah&gt;I&#39;m alright.&lt;/speech&gt;
 
-&quot;I&#39;m alright&quot;, she replies.
+&lt;action&gt;She replies.&lt;/action&gt;
 
-I let out a breath, alright, now to figure out how to get me down there.
+&lt;action&gt;I let out a breath, alright, now to figure out how to get me down there.&lt;/action&gt;
 
 &lt;% if (!window.story.getChoice(&quot;Chapter1&quot;, &quot;StrangerDied&quot;)) { %&gt;
 
-&quot;Can you lower me down?&quot;, the stranger asks.
+&lt;resetcharacter.two/&gt;
+&lt;character.two&gt;Unknown&lt;/character&gt;
+
+&lt;speech.unknown&gt;Can you lower me down?&lt;/speech&gt;
+
+&lt;action&gt;the stranger asks.&lt;/action&gt;
 
-&quot;What? No!&quot;, I snap back.
+&lt;speech.sarah&gt;What? No!&lt;/speech&gt;
+
+&lt;action&gt;I snap back.&lt;/action&gt;
 
 &lt;% } %&gt;
 
-[[[Stand up]|C1P61]]</tw-passagedata><tw-passagedata pid="286" name="C1P61" tags="page" position="3075,15650" size="100,100">&lt;% window.story.delayedText(15 * 1000) %&gt;
+[[[Stand up]|C1P61]]</tw-passagedata><tw-passagedata pid="286" name="C1P61" tags="page chase" position="3075,15650" size="100,100">&lt;ui&gt;standard&lt;/ui&gt;
+&lt;character.one.pain.neutral.neutral-backpack&gt;Sarah&lt;/character&gt;
 
-I get back up and step away from the edge.
 
-Alright, maybe if-
+&lt;action&gt;I get back up and step away from the edge.&lt;/action&gt;
+&lt;action&gt;Alright, maybe if-&lt;/action&gt;
 
 &lt;% if (!window.story.getChoice(&quot;Chapter1&quot;, &quot;StrangerDied&quot;)) { %&gt;
 
-&quot;Shit, watch out!&quot;, the stranger shouts.
+&lt;character.two&gt;Unknown&lt;/character&gt;
+
+&lt;speech.unknown&gt;Shit, watch out!&lt;/speech&gt;
+
+&lt;action&gt;the stranger shouts.&lt;/action&gt;
 
 &lt;% } %&gt;
 
-&quot;Argh!&quot;, I let out.
+&lt;speech.sarah&gt;Argh!&lt;/speech&gt; 
+&lt;action&gt;I let out.&lt;/action&gt;
```

Comment:
Another example where it might read better to keep; but could also use a tweak, maybe:

```suggestion
&lt;action&gt;I instinctively cry out.&lt;/action&gt;
```